### PR TITLE
Remove JSON roundtripping part

### PR DESCRIPTION
**Pull request by rricard**
cc. @littledan & @mheiber

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,9 +172,7 @@ At runtime, if a non-value type is placed inside a `Record` or `Tuple`, it is a
 
 At runtime, attempting to create a `Record` with a key that is not a `string` is a `TypeError`.
 
-At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `true`, `false`, and `null`.
-
-> `undefined` is not permitted, as that would violate the goal of lossless round-tripping with JSON.
+At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `true`, `false`, `undefined` and `null`.
```
Comment (robpalme) [inline README.md]: Should also permit symbol

Comment (rricard) [reply README.md]: yup, thanks!

File: README.md [hunk 2]
```diff
@@ -174,9 +172,7 @@ At runtime, if a non-value type is placed inside a `Record` or `Tuple`, it is a
 
 At runtime, attempting to create a `Record` with a key that is not a `string` is a `TypeError`.
 
-At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `true`, `false`, and `null`.
-
-> `undefined` is not permitted, as that would violate the goal of lossless round-tripping with JSON.
+At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `symbol`, `true`, `false`, `undefined` and `null`.
```
Comment (ljharb) [inline README.md]: true and false aren’t types, do you mean “boolean”?

File: README.md (hunk 1 above)
Comment (littledan) [reply README.md]: and BigInt?

File: README.md (hunk 2 above)
Comment (rricard) [reply README.md]: Agreed, we meant `boolean`

Comment (rricard) [reply README.md]: updated on master

**Pull request merged**